cf_xarray/accessor.py

```diff
@@ -65,6 +65,16 @@
 ATTRS["time"] = ATTRS["T"]
 ATTRS["vertical"] = ATTRS["Z"]
 
+OPTIONS: MutableMapping[str, Any] = {"custom_criteria": []}
+
+
+def set_options(custom_criteria):
+    OPTIONS["custom_criteria"] = (
+        always_iterable(custom_criteria, allowed=(tuple, list, set))
+        + OPTIONS["custom_criteria"]
+    )
```

Comment:
```suggestion
    OPTIONS["custom_criteria"] = always_iterable(custom_criteria, allowed=(tuple, list, set))
```

How about we do this for now, so we can always reset it?

Reply:
Sounds good!